app/models/doctor_appointment_time.rb

```diff
@@ -0,0 +1,5 @@
+class DoctorAppointmentTime < ApplicationRecord
+  has_many :doctor_times
+  belongs_to :doctor
+  # belongs_to :appointment
+end
```

Comment:
- [ ] Please ensure that the models have proper validation to prevent the storage of inaccurate data in the database.
- [ ] Kindly remove the commented code.